src/scripts/main.js

```diff
@@ -1 +1,106 @@
 'use strict';
+
+const bodyDoc = document.querySelector('body');
+let rightClick;
+let leftClick;
+
+function makeDiv(className) {
+  const div = document.createElement('div');
+
+  div.setAttribute('data-qa', 'notification');
+  div.setAttribute('class', className);
+  bodyDoc.appendChild(div);
```

Comment:
The function `makeDiv` creates a new div and appends it to the body each time it's called. If it's called multiple times without any clean-up, it could lead to a large number of div elements being added to the DOM, which can affect performance. Consider implementing a clean-up mechanism if necessary.